3.10/aql/functions-miscellaneous.md

```diff
@@ -158,6 +158,16 @@ a request context. Otherwise, the return value of this function will be *null*.
 - returns **userName** (string\|null): the current user name, or *null* if
   authentication is disabled
 
+### CURRENT_DATABASE()
+
+`CURRENT_DATABASE() → databaseName`
+
+Returns the name of the current database.
+
+The current database is the database name that was specified while loging in.
```

Comment:
This should say that the database is specified in the URL path of the request (or defaults to _system database).

E.g. an AQL query execution in the context of database db1 would be done by using the endpoint `http://host/_db/db1/_api/cursor`